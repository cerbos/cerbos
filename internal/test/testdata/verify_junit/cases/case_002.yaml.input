-- suite_test.yaml --
---
name: CompanyTestSuite
description: Tests for verifying the company resource policy
skip: true
principals:
  admin:
    id: admin
    roles:
      - admin
  user:
    id: user
    roles:
      - user
resources:
  company:
    id: xx1
    kind: company

tests:
  - name: Company Test 1
    input:
      principals:
        - admin
      resources:
        - company
      actions:
        - create
    expected:
      - principal: admin
        resource: company
        actions:
          create: EFFECT_DENY
