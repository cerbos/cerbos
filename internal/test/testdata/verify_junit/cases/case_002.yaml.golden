<testsuites errors="0" failures="0" skipped="1" tests="0">
  <testsuite description="Tests for verifying the company resource policy" name="CompanyTestSuite" file="suite_test.yaml" errors="0" failures="0" skipped="1" tests="0">
    <skipped message="This test suite was skipped"></skipped>
    <properties></properties>
  </testsuite>
</testsuites>